src/connection.ts

```diff
@@ -492,6 +491,8 @@ export class StableWSConnection<
         return;
       } else {
         this.resolvePromise?.(event);
+        // set healthy..
+        this._setHealth(true);
```

Comment:
this should fix the bug "watch or presence requires an active WebSocket connection"